docs/DG_Diagrams/RouteCommandClass.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
+
+abstract class Command
+Class RouteCommand
+Class RouteParser
+Class Parser
+Class BusData {
+{static} possibleBuses (:String, :String) : ArrayList<Bus>
+}
+Class Bus {
+getPossibleRoute(:String, :String) : ArrayList<BusStops>
+}
+Class SimilarityCheck {
+{static} similarLoc(:String) : ArrayList<String>
+}
+
+Class Ui {
+{static} printRouteMessage(:ArrayList<Bus>)
+{static} printPossibleLocsMessage(:ArrayList<String>)
+}
+
+enum BusStops {
```

Comment:
![e2](https://user-images.githubusercontent.com/60377900/97544883-2f8f2100-1a05-11eb-9e42-d7a857e4d5e8.png)

Maybe can consider adding types of attributes to the enum